.github/workflows/scripts/install.sh

```diff
@@ -75,9 +75,6 @@ services:
     image: "pulp:${TAG}"
     volumes:
       - ./settings:/etc/pulp
-  - name: ciproxy
-    image: ghcr.io/abhinavsingh/proxy.py:latest
-    command: "--basic-auth foo:bar --hostname 0.0.0.0"
```

Comment:
Is this the current state of the plugin template? I'd hate to do ping pong with the pulp bot.

Reply:
Ahh, ty. I'll rebase onto it now.